depmapomics/constants.py

```diff
@@ -295,60 +293,115 @@
 
 HESS_COL = "HessDriver"
 
+ONCOKB_HOTSPOT_COL = "OncoKBHotspot"
+
+COSMIC_TIER_COL = "CosmicTier"
+
+PROTEIN_DICT = {'Cys': 'C', 'Asp': 'D', 'Ser': 'S', 'Gln': 'Q', 'Lys': 'K',
+     'Ile': 'I', 'Pro': 'P', 'Thr': 'T', 'Phe': 'F', 'Asn': 'N', 
+     'Gly': 'G', 'His': 'H', 'Leu': 'L', 'Arg': 'R', 'Trp': 'W', 
+     'Ala': 'A', 'Val':'V', 'Glu': 'E', 'Tyr': 'Y', 'Met': 'M'}
+
 MUTCOL_DEPMAP = {
-    "chrom": "Chrom",
-    "pos": "Pos",
-    "ref": "Ref",
-    "alt": "Alt",
-    "af": "AF",
-    "ref_count": "RefCount",
-    "alt_count": "AltCount",
-    "gt": "GT",
-    "ps": "PS",
-    "variant_type": "VariantType",
-    "variant_info": "VariantInfo",
-    "dna_change": "DNAChange",
-    "protein_change": "ProteinChange",
-    "hugo_symbol": "HugoSymbol",
-    "hgnc_name": "HgncName",
-    "hgnc_family": "HgncFamily",
-    "transcript": "Transcript",
-    "transcript_exon": "TranscriptExon",
-    "transcript_strand": "TranscriptStrand",
-    "uniprot_id": "UniprotID",
-    "str": "Str",
-    "dbsnp_id": "DbsnpID",
-    "dbsnp_filter": "DbsnpFilter",
-    "issues": "Issues",
-    "gc_content": "GcContent",
-    "lineage_association": "LineageAssociation",
-    "cancer_molecular_genetics": "CancerMolecularGenetics",
-    "ccle_deleterious": "CCLEDeleterious",
-    "structural_relation": "StructuralRelation",
-    "cosmic_hotspot": "CosmicHotspot",
-    "cosmic_overlapping_mutations": "CosmicOverlappingMutations",
-    "associated_with": "AssociatedWith",
-    "lof": "LoF",
-    "driver": "Driver",
-    "likely_driver": "LikelyDriver",
-    "transcript_likely_lof": "TranscriptLikelyLoF",
-    "civic_id": "CivicID",
-    "civic_description": "CivicDescription",
-    "civic_score": "CivicScore",
-    "popaf": "Popaf",
-    "likely_gof": "LikelyGoF",
-    "likely_lof": "LikelyLoF",
-    "hess_driver": "HessDriver",
-    "hess_signture": "HessSignature",
-    "hess_signature": "HessSignature",
-    "revel_score": "RevelScore",
-    "pharmgkb_id": "PharmgkbID",
-    "dida_id": "DidaID",
-    "dida_name": "DidaName",
-    "gwas_disease": "GwasDisease",
-    "gwas_pmid": "GwasPmID",
-    "gtex_gene": "GTexGene",
-    "DepMap_ID": "DepMap_ID",
+    'chrom': 'Chrom',
```

Comment:
@qinqian can you take a look at these two lists of columns to include? I've commented out things that are in the sample-level maf that I think we don't need for the release. Let me know what you think!